docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -586,56 +586,56 @@ For Tensorflow:
+It means that your mean/scale values have a wrong format. Specify mean/scale values using the form `layer_name(val1,val2,val3)`.
+You need to specify values for each input of the model. For more information, refer to the [Converting a Model to Intermediate Representation](convert_model/Converting_Model.md) guide.
 
-#### 102. What does the message "Operation _contrib_box_nms is not supported ..." mean? <a name="question-102"></a>
+#### 98. What does the message "Operation _contrib_box_nms is not supported ..." mean? <a name="question-98"></a>
 
-It means that you trying to convert the topology which contains '_contrib_box_nms' operation which is not supported directly. However the sub-graph of operations including the '_contrib_box_nms' could be replaced with DetectionOutput layer if your topology is one of the gluoncv topologies. Specify '--enable_ssd_gluoncv' command line parameter for the Model Optimizer to enable this transformation.
+It means that you are trying to convert the topology, which contains `_contrib_box_nms` operation which is not supported directly. However, the sub-graph of operations including the `_contrib_box_nms` could be replaced with DetectionOutput layer if your topology is one of the gluoncv topologies. Specify `--enable_ssd_gluoncv` command line parameter for Model Optimizer to enable this transformation.
 
-#### 103. What does the message "ModelOptimizer is not able to parse *.caffemodel" mean? <a name="question-103"></a>
+#### 99. What does the message "ModelOptimizer is not able to parse *.caffemodel" mean? <a name="question-99"></a>
 
-If a '*.caffemodel' file exists and it is correct, the error possibly occured due to the use of Python protobuf implementation. In some cases, it shows error message during model parsing, for example: "'utf-8' codec can't decode byte 0xe0 in position 4: invalid continuation byte in field: mo_caffe.SpatialTransformerParameter.transform_type". You can either use Python 3.6/3.7 or build 'cpp' implementation of protobuf yourself for your version of Python. For the complete instructions about building `protobuf` from sources, see the appropriate section in [Converting a Model to Intermediate Representation](../Deep_Learning_Model_Optimizer_DevGuide.md).
+If a `*.caffemodel` file exists and it is correct, the error possibly occurred due to the use of Python protobuf implementation. In some cases, it shows error message during model parsing, for example: "`utf-8` codec can't decode byte 0xe0 in position 4: invalid continuation byte in field: mo_caffe.SpatialTransformerParameter.transform_type". You can either use Python 3.6/3.7 or build `cpp` implementation of protobuf yourself for your version of Python. For the complete instructions about building `protobuf` from sources, see the appropriate section in the [Converting a Model to Intermediate Representation](../Deep_Learning_Model_Optimizer_DevGuide.md) guide.
```

Comment:
```suggestion
If a `*.caffemodel` file exists and is correct, the error  occurred possibly because of the use of Python protobuf implementation. In some cases, error messages may appear during model parsing, for example: "`utf-8` codec can't decode byte 0xe0 in position 4: invalid continuation byte in field: mo_caffe.SpatialTransformerParameter.transform_type". You can either use Python 3.6/3.7 or build the `cpp` implementation of `protobuf` yourself for your version of Python. For the complete instructions about building `protobuf` from sources, see the appropriate section in the [Converting Models with Model Optimizer](../Deep_Learning_Model_Optimizer_DevGuide.md) guide.
```